policies/releasestrat.html

```diff
@@ -88,9 +77,9 @@ <h5>
 	    <li>3rd April 2018, beta release 2 (pre4)</li>
 	    <li>17th April 2018, beta release 3 (pre5)</li>
 	    <li>1st May 2018, beta release 4 (pre6)</li>
-	    <li>8th May 2018, release readiness check (new release
-		cycles added if required, first possible final release date:
-		15th May 2018)</li>
+            <li>8th May 2018 (pre7)</li>
+            <li>29th May 2018 (pre8)</li>
+            <li>19th June 2018 (pre9)</li>
```

Comment:
8th May was not a release: it was a release readiness check (which we effectively did a few days early at the f2f). The last release was pre6. The other releases are called "beta release 2" etc, but the one's you added don't have that. We should be consistent.

Again the vote that backs up this change is still open, so this should be pushed until after it has closed.